docs/UserGuide.md

```diff
@@ -450,6 +450,17 @@ Format: `reminder delete INDEX`
 Examples:
 * `reminder list` followed by `reminder delete 2` deletes the 2nd reminder in StonksBook.
 
+#### Filtering reminders: `reminder find` \[Wang Luo\]
+
+Finds the reminders displayed based on the completion status.
```

Comment:
Maybe can add a note here that the default reminder list is based on `reminder find st/pending`?